netty/src/test/java/io/grpc/netty/NettyClientHandlerTest.java

```diff
@@ -217,6 +219,32 @@ public Void answer(InvocationOnMock invocation) throws Throwable {
     channelRead(serializedSettings);
   }
 
+  @Test
+  @SuppressWarnings("InlineMeInliner")
```

Comment:
What was wanting to be inlined?

Reply:
The inliner wants me to use `string.repeat()` instead of `Strings.repeat()` but that method is JDK 11+. I am not sure we want to use JDK 11 methods even for tests, so I suppressed the warning.

Reply:
:-/ . That sort of thing is going to get more annoying. But ignoring it is right. We'll often put a comment on the line with the annotation (e.g., `// String.repeat() requires Java 11`) to make clear why it is there.